### PR TITLE
make pydap canonical project name

### PR DESCRIPTION
**Pull request by tomkralidis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 [![Python2](https://img.shields.io/badge/python-2-blue.svg)](https://www.python.org/downloads/)
 [![Python3](https://img.shields.io/badge/python-3-blue.svg)](https://www.python.org/downloads/)
 [![documentation](https://readthedocs.org/projects/pydap/badge/?version=latest)](http://pydap.readthedocs.org/en/latest/)
-[![PyPI](https://img.shields.io/pypi/v/pydap.svg?maxAge=2592000?style=plastic)](https://pypi.python.org/pypi/Pydap/)
+[![PyPI](https://img.shields.io/pypi/v/pydap.svg?maxAge=2592000?style=plastic)](https://pypi.python.org/pypi/pydap/)
 [![Join the chat at https://gitter.im/pydap/pydap](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/pydap/pydap) 
 
-[Pydap](http://pydap.readthedocs.io/en/latest/) is an implementation of the Opendap/DODS protocol, written from scratch in pure python.  You can use Pydap to access scientific data on the internet without having to 
+[pydap](http://pydap.readthedocs.io/en/latest/) is an implementation of the Opendap/DODS protocol, written from scratch in pure python.  You can use pydap to access scientific data on the internet without having to
```
Comment (7yl4r) [inline README.md]: Not sure about usage at the start of a sentence. I have no preference, but @jhamman suggested capitalizing at start like with `xarray`.

File: docs/client.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Using the client
 ================
 
-Pydap can be used as a client to inspect and retrieve data from any of the `hundreds of scientific datasets <http://www.opendap.org/data/datasets.cgi?xmlfilename=datasets.xml&exfunction=none>`_ available on the internet on `OPeNDAP <http://opendap.org/>`_ servers. This way, it's possible to instrospect and manipulate a dataset as if it were stored locally, with data being downloaded on-the-fly as necessary.
+pydap can be used as a client to inspect and retrieve data from any of the `hundreds of scientific datasets <http://www.opendap.org/data/datasets.cgi?xmlfilename=datasets.xml&exfunction=none>`_ available on the internet on `OPeNDAP <http://opendap.org/>`_ servers. This way, it's possible to instrospect and manipulate a dataset as if it were stored locally, with data being downloaded on-the-fly as necessary.
```
Comment (7yl4r) [inline docs/client.rst]: another sentence starter here.

File: docs/client.rst [hunk 3]
```diff
@@ -298,7 +298,7 @@ You can also access the deep variables directly. When you iterate over these var
     [5.0, 10.0, 15.0, 20.0, 25.0, 30.0, 35.0, 40.0, 45.0, 50.0]
     [5.0999999, 9.1000004, 19.4, 29.700001, 39.599998, 49.599998, 59.700001, 69.5, 79.5, 89.699997]
 
-Pydap 3.0 has been rewritten to make it easier to work with Dapper datasets like this one, and it should be intuitive [1]_ to work with these variables. 
+pydap 3.0 has been rewritten to make it easier to work with Dapper datasets like this one, and it should be intuitive [1]_ to work with these variables.
```
Comment (7yl4r) [inline docs/client.rst]: and another!

File: docs/developer_templating.rst [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Templating
 ----------
 
-Pydap uses an `experimental backend-agnostic templating API <http://svn.pythonpaste.org/Paste/TemplateProposal/>`_ for rendering HTML and XML by responses. Since the API is neutral Pydap can use any templating engine, like `Mako <http://www.makotemplates.org/>`_ or `Genshi <http://genshi.edgewall.org/>`_, and templates can be loaded from disk, memory or a database. The server that comes with Pydap, for example, defines a ``renderer`` object that loads Genshi templates from a directory ``templatedir``:
+pydap uses an `experimental backend-agnostic templating API <http://svn.pythonpaste.org/Paste/TemplateProposal/>`_ for rendering HTML and XML by responses. Since the API is neutral pydap can use any templating engine, like `Mako <http://www.makotemplates.org/>`_ or `Genshi <http://genshi.edgewall.org/>`_, and templates can be loaded from disk, memory or a database. The server that comes with pydap, for example, defines a ``renderer`` object that loads Genshi templates from a directory ``templatedir``:
```
Comment (7yl4r) [inline docs/developer_templating.rst]: 4th sentence start

File: docs/index.rst [hunk 5]
```diff
@@ -2,21 +2,21 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Pydap
+pydap
 =====
 
-Pydap is a pure `Python <http://python.org/>`_ library implementing the `Data Access Protocol <http://opendap.org/>`_, also known as **DODS** or **OPeNDAP**. You can use Pydap as a `client <client.html>`_ to access hundreds of scientific datasets in a transparent and efficient way through the internet; or as a server to easily `distribute <server.html>`_ your data from a `variety of formats <handlers.html>`_.
+pydap is a pure `Python <http://python.org/>`_ library implementing the `Data Access Protocol <http://opendap.org/>`_, also known as **DODS** or **OPeNDAP**. You can use pydap as a `client <client.html>`_ to access hundreds of scientific datasets in a transparent and efficient way through the internet; or as a server to easily `distribute <server.html>`_ your data from a `variety of formats <handlers.html>`_.
```
Comment (7yl4r) [inline docs/index.rst]: 5th sentence start & header above (I think header should remain `pydap`)

File: README.md (hunk 1 above)
Comment (jhamman) [reply README.md]: I don't have super strong feelings on how this should be done. I was just offering a potential path forward given our recent experience. It always strikes me as odd when sentences start with a lower case letter 😉.

File: docs/developer_templating.rst (hunk 4 above)
Comment (tomkralidis) [reply docs/developer_templating.rst]: I'd prefer we keep `pydap` as `pydap` regardless of start of sentence, to keep things clear?

File: docs/index.rst (hunk 5 above)
Comment (tomkralidis) [reply docs/index.rst]: I'd prefer we keep pydap as pydap regardless of start of sentence, to keep things clear?

**Pull request merged**